include/raft.h

```diff
@@ -264,8 +269,9 @@ struct raft_append_entries_result
     raft_term term;            /* Receiver's current_term. */
     raft_index rejected;       /* If non-zero, the index that was rejected. */
     raft_index last_log_index; /* Receiver's last log entry index, as hint. */
+    raft_flags flags;          /* Feature flags. */
```

Comment:
Perhaps it would help a bit if we rename this field from `flags` to `features`? The word "flags" makes me think to specific options that apply the message. Renaming it "features" would make it a bit more explicit that this field contains the features the sending node knows about.

We might actually add a real message-specific "flags" in the future that is separate from the "features". Also, the `RequestVote` encoding already uses a "flags" field (see [here](https://github.com/canonical/raft/blob/529c5c83f44f32432131c43eee5b7bfd27a1df4d/src/uv_encoding.c#L90)) to encode message-specific flags. If we were to add the "features" field to also other messages in the future, this rename would avoid a clash, at least in terms of internal code/encoding.